docs/compression_algorithms/CompressWeights.md

```diff
@@ -289,9 +289,9 @@ Here is the perplexity and model size before and after weight compression for di
 
 Here is the word perplexity with data-free and data-aware mixed-precision INT4-INT8 weight compression for different language models on the [wikitext dataset](https://arxiv.org/pdf/1609.07843.pdf).
 `data` suffix refers to the data-aware mixed-precision.
-
+`data_awq` suffix refers to the data-aware mixed-precision with modified [AWQ](https://arxiv.org/abs/2306.00978) algorithm.
```

Comment:
Maybe it makes sense to add somewhere about specific pattern of layers in order to apply AWQ.

Reply:
Done